css/accordion.css

```diff
@@ -71,6 +71,13 @@ html {
   padding: 0.5rem 1.2rem;
 }
 
+.acc input:checked ~ label .rotate {
```

Comment:
Need to check browser compatibility of this selector. There might be a simpler one that fits just as well, too.